examples/hello-compute/README.md

```diff
@@ -0,0 +1,30 @@
+# hello-compute
+
+Runs a compute shader to determine the number of iterations of the rules from
+Collatz Conjecture
+
+- If n is even, n = n/2
+- If n is odd, n = 3n+1
+
+that it will take to finish and reach the number `1`.
+
+## To Run
+
+```
+# Pass in any 4 numbers as arguments
+RUST_LOG=info cargo run --example hello-compute 1 2 3 4
```

Comment:
We could limit the log info to `hello_compute` only by using something like `RUST_LOG=hello_compute cargo run --example hello-compute 1 2 3 4`, I think this would make the output clearer

Reply:
Didn’t even know you could do that thanks - will do